codebuild_specs/aggregate_e2e_reports.yml

```diff
@@ -11,12 +11,17 @@ phases:
       - cd ..
       - source ./shared-scripts.sh && _downloadReportsFromS3 $CODEBUILD_SOURCE_VERSION
       - mkdir artifacts && echo "empty artifact" > artifacts/empty-artifact.txt
+  post_build:
+    commands:
+      - echo $PROJECT_NAME
```

Comment:
I had to write the repetitive code because after I used shared_script.sh in f0a4fcc44a7f666c1c431c9a9b4bad7e89b147f6, some builds couldn't find the file.